### PR TITLE
Risk-based execution

### PR DESCRIPTION
**Pull request by mikimandoki**
- Extended the `androidIt`, `iosIt`, `mobileIt`, `bothPlatformsIt` functions to expect one of four mandatory testRisk values 
   - These are 'high', 'medium', 'low', and 'undefined'
   - They are passed along as Playwright @ tags in the test names 
- Added the execution scripts `test-high-risk-android` and `test-high-risk-ios` to run only `@high-risk` tests per platform
- Assigned a tentative risk value to every test case 
   - Currently there are 29 high-risk test cases (out of 71) which is a 60% reduction  
- Consolidated identical `androidIt` and `iosIt` function calls to use `bothPlatformsIt` instead

### CODE REVIEW

**Review by burtonemily — APPROVED** [review]
Looks good to me

File: run/types/testing.ts [hunk 1]
```diff
@@ -408,4 +409,9 @@ export type Id =
   | 'network.loki.messenger:id/openGroupTitleTextView'
   | 'Image picker'
   | 'network.loki.messenger:id/action_apply';
-  | 'Error message';
+
+export type testRisk =
```
Comment (Bilb) [inline run/types/testing.ts:413]: this should be capitalized `TestRisk` as it is a type and not a variable name

File: run/types/sessionIt.ts [hunk 2]
```diff
@@ -56,10 +58,11 @@ export function iosIt(
 function mobileIt(
   platform: SupportedPlatformsType,
   title: string,
+  testRisk: testRisk,
   testToRun: (platform: SupportedPlatformsType) => Promise<void>,
   shouldSkip: boolean = false
 ) {
-  const testName = `${title} ${platform}`;
+  const testName = `${title} ${platform} @${testRisk}-risk`;
```
Comment (Bilb) [inline run/types/sessionIt.ts:65]: why not use playwright annotations? https://playwright.dev/docs/test-annotations#tag-tests

Comment (mikimandoki) [reply run/types/sessionIt.ts:65]: I thought the @-token in the test title acts as a test tag?

File: run/types/testing.ts (hunk 1 above)
Comment (mikimandoki) [reply run/types/testing.ts:413]: fixed this in https://github.com/oxen-io/session-appium/pull/18

File: run/types/sessionIt.ts (hunk 2 above)
Comment (Bilb) [reply run/types/sessionIt.ts:65]: yep all good, I didn't know playwright was scanning the testName for any @ and using it as tags

**Pull request merged**